src/test/java/org/painandsuffer/items/armour/chest/ChainmailTest.java

```diff
@@ -0,0 +1,34 @@
+package org.painandsuffer.items.armour.chest;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ChainmailTest {
+    @Test
+    public void whenChainmailCreatedWithDefaultDefendAndArmourDurability_thenChainmailCreatedWithDefend15AndArmourDurability100(){
+        Chainmail chainmail = new Chainmail();
+        int chainmailDefend = 15;
+        int chainmailDurability = 100;
+        assertNotNull(chainmail);
+        assertEquals(chainmail.getDefence(),chainmailDefend);
+        assertEquals(chainmail.getArmourDurability(),chainmailDurability);
+    }
+
+    @Test
+    public void whenChainmailCreatedWithDefaultAndOverride_thenChainmailHasNewDefenceAndArmourDurability(){
+        Chainmail chainmail = new Chainmail();
+        chainmail.setDefence(10);
+        chainmail.setArmourDurability(150);
+        assertNotNull(chainmail);
+        assertEquals(chainmail.getDefence(),10);
+        assertEquals(chainmail.getArmourDurability(),150);
+    }
+
+    @Test
+    public void whenChainmailUseDescription_thenDescriptionAppears(){
```

Comment:
Applicable to all Armour test classes.
Test verify that the description method returns a non-null string. However, this test don't assert anything about the content of the description, which means they could be improved to ensure the description is meaningful and correct.